docs/pages/docs/core-concepts/caching.mdx

```diff
@@ -190,11 +190,11 @@ Note that this automatic detection and inclusion only works if Turborepo success
   cache. To workaround this, Turborepo uses a `TURBO_CI_VENDOR_ENV_KEY` variable to
   _exclude_ environment variables from Turbo's inference.
 
-  For example, Vercel sets the `NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA`. This value changes on every build,
-  so Vercel _also_ sets `TURBO_CI_VENDOR_ENV_KEY="NEXT_PUBLIC_VERCEL_"` to exclude these variables.
+For example, Vercel sets the `NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA`. This value changes on every build,
+so Vercel _also_ sets `TURBO_CI_VENDOR_ENV_KEY="NEXT_PUBLIC_VERCEL_"` to exclude these variables.
```

Comment:
unindent here seems unintentional? it's a `<Callout>` block

Reply:
Prettier is telling me this is OK, so I'll trust it.